src/test/model-viewer-base-spec.ts

```diff
@@ -208,6 +208,46 @@ suite('ModelViewerElementBase', () => {
           expect(dataUrlMatcher.test(element.toDataURL())).to.be.true;
         });
       });
+
+      suite('toBlob', () => {
+        test('produces a blob', async () => {
+          const blob = await element.toBlob();
+          expect(blob).to.not.be.null;
+        });
+
+        test('can convert blob to object URL', async () => {
+          const blob = await element.toBlob();
+          const objectUrl = URL.createObjectURL(blob);
+          const objectUrlMatcher = /^blob\:https?:\//;
+          expect(objectUrlMatcher.test(objectUrl)).to.be.true;
+        });
+
+        test('has size', async () => {
+          const blob = await element.toBlob();
+          expect(blob.size).to.be.greaterThan(0);
+        });
+
+        test('uses fallbacks on unsupported browsers', async () => {
+          // Emulate unsupported browser
+          const restoreCanvasToBlob = spy(HTMLCanvasElement.prototype, 'toBlob', { value: undefined });
```

Comment:
This was intended to help us emulated unsupported browsers. Unfortunately, in Edge this had an unintended consequence (throwing when we can't find the property being spied on).

I would wrap this call in a try/catch, and ignore exceptions thrown here.

Reply:
Alternatively, we could change `spy` to define properties even if they are missing. I would accept a change that causes `spy` to define the property on the originally-passed-in object when no property descriptor for the property is found.

In this case, when "restoring" the property, we should effectively delete it from the object.

Reply:
This seems reasonable, currently I have the `try/catch` in place but if we decide to implement this I can update these tests.

Reply:
`try/catch` is fine for now, we can do the other change another time.